arm-network/2016-09-01/swagger/routeFilter.json

```diff
@@ -0,0 +1,849 @@
+        "description": "Get the available service communities.",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the service community."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of ExpressRouteServiceProdiver resources.",
+            "schema": {
+              "$ref": "#/definitions/ServiceCommunity"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/serviceCommunities": {
+      "get": {
+        "tags": [
+          "ServiceCommunities"
+        ],
+        "operationId": "ServiceCommunities_List",
+        "description": "Gets all the available service communities.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of ServiceCommunity resources.",
+            "schema": {
+              "$ref": "#/definitions/ServiceCommunityListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RouteFilterRulePropertiesFormat": {
+      "properties": {
+        "access": {
+          "type": "string",
+          "description": "The access type of the rule. Possible values are: 'Allow', 'Deny",
+          "enum": [
+            "Allow",
+            "Deny"
+          ],
+          "x-ms-enum": {
+            "name": "Access",
+            "modelAsString": true
+          }
+        },
+        "routeFilterRuleType": {
+          "type": "string",
+          "description": "The type of the rule. Possible values are: 'Community",
+          "enum": [
+            "Community"
+          ],
+          "x-ms-enum": {
+            "name": "RouteFilterRuleType",
+            "modelAsString": true
+          }
+        },
+        "communities": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The collection for bgp communities to filter on."
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "The provisioning state of the resource. Possible values are: 'Updating', 'Deleting', 'Succeeded' and 'Failed'."
+        }
+      },
+      "required": [
+        "nextHopType"
```

Comment:
Even the existing network spec, has a similar same issue. required array has the name nextHopType but the model had nextHop in that case. Please make sure that required array has model properties accurately defined.